Form/Type/ModelType.php

```diff
@@ -51,7 +51,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         if ($options['multiple']) {
             if (array_key_exists('choice_loader', $options) && $options['choice_loader'] !== null) { // SF2.7+
-                $builder->addViewTransformer(new ModelsToArrayTransformer($options['choice_list'], $options['model_manager'], $options['class']), true);
```

Comment:
this is only a formatting change, isn't it? it confused me a little bit, can we do this in a separate PR?